Dockerfile

```diff
@@ -1,24 +1,21 @@
 ARG PHP_VERSION=8.1
-ARG COMPOSER_VERSION=2.5.4
 
 FROM php:${PHP_VERSION}-cli
 
 RUN apt-get update && \
-    apt-get install -y autoconf pkg-config libssl-dev git libzip-dev zlib1g-dev && \
+    apt-get install -y autoconf pkg-config libssl-dev git unzip libzip-dev zlib1g-dev && \
     pecl install mongodb && docker-php-ext-enable mongodb && \
     pecl install xdebug && docker-php-ext-enable xdebug && \
     docker-php-ext-install -j$(nproc) pdo_mysql zip
 
-COPY --from=composer:${COMPOSER_VERSION} /usr/bin/composer /usr/local/bin/composer
+COPY --from=composer:2.5.8 /usr/bin/composer /usr/local/bin/composer
```

Comment:
Having it at the top of the file made it easier to locate the version for an update, but no objection to inlining it.

Reply:
It doesn't work with my setup.
```
Docker version 24.0.5, build ced0996600
Docker Compose version 2.20.3
colima version 0.5.5
```

There is a workaroud if I alias the composer image.
https://github.com/docker/for-mac/issues/2155#issuecomment-338619432